Directory.Build.props

```diff
@@ -24,10 +24,10 @@
     <!-- The windows SDK version of UWP, Modern UWP and WinUI could be aligned -->
     <UwpMinimum>uap10.0.16299</UwpMinimum>
     <ModernUwpMinimum>net9.0-windows10.0.17763.0</ModernUwpMinimum>
-    <WinUiMinimum>net6.0-windows10.0.18362.0</WinUiMinimum>
+    <WinUiMinimum>net8.0-windows10.0.18362.0</WinUiMinimum>
 
-    <MicrosoftTestingTargetFrameworks>net6.0;net7.0;net8.0;net9.0</MicrosoftTestingTargetFrameworks>
-    <SupportedNetFrameworks>netcoreapp3.1;net6.0;net7.0;net8.0;net9.0</SupportedNetFrameworks>
+    <MicrosoftTestingTargetFrameworks>net8.0;net9.0</MicrosoftTestingTargetFrameworks>
+    <SupportedNetFrameworks>net8.0;net9.0</SupportedNetFrameworks>
```

Comment:
Could we do a follow-up PR merging these 2 set of properties?

Reply:
@Evangelink, sure. This assumes that MSTest will always be matching the TFMs of MTP though. Do we want to make that assumption going forward?

Reply:
We can introduce back 2 properties if that were to happen. Given we support what's supported by MS, I feel like the .NET version would always match.